rules/order.js

```diff
@@ -30,6 +55,7 @@ module.exports = {
         'box-sizing',
         'display',
         'overflow',
+        'vertical-align',
```

Comment:
I personally like overflow after width and height, flows logically after specifying dimensions.